docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -0,0 +1,310 @@
+This function is useful for setting up global state or a server that will be used multiple journeys.
+* `before(function)` -- Runs a provided function prior to a single `journey` runs.
+This function is useful for setting up global state or a server that will be used for a single journey.
+* `afterAll(function)` -- Runs a provided function after all the journeys have completed.
+This function is useful for removing global state or closing a server that was used in multiple journeys.
+* `after(function)` -- Runs a provided function after a single journey has completed.
+This function is useful for removing global state or closing a server that was used in a single journey.
+
+TIP: {playwright-url}[Playwright] is browser testing library developed by Microsoft.
+It is reliable and fast and features a modern API that auto waits for page elements to be ready.
+
+[discrete]
+[[synthetics-create-journey]]
+=== Create a `journey`
+
+The `journey` function takes two main parameters, `page` and `params`:
+
+* `page` -- A `page` object from Playwright that lets you control the browser's current page.
+* `params` -- User-defined variables that allow you to invoke the Synthetics suite with custom parameters.
+For example, if you want to use a different homepage depending on the `env`
+(localhost for `dev`, and a URL for `prod`).
+
+Putting it all together, a basic, two step journey might look something like this:
+
+[source,js]
+----
+journey("Journey title", async ({page}) => {
+    step("Step 1 title", async () => {
+      // Do something here
+    })
+    step("Step 2 title", async () => {
+      // Do something else here
+    })
+});
+----
+
+// REVIEWERS
+// Should we add a note here about the `async` keyword / promises / link to mdn docs?
+// https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await[async/await].
+
+[discrete]
+[[synthetics-create-step]]
+=== Create a `step`
+
+Steps can be as simple or complex as you need them to be.
+The {playwright-api-docs}[Playwright page API reference] will be your friend while writing tests.
+
+A basic first step might simply load a page:
+
+[source,js]
+----
+await page.goto('https://www.elastic.co'); <1>
+----
+<1> See the https://github.com/microsoft/playwright/blob/master/docs/api.md#pagegotourl-options[`page.goto` reference] for more information.
+
+A more complex second step might wait for a page element to be selected,
+and then ensure that it matches an expected value.
+Consider the following HTML snippet:
+
+[source,html]
+----
+<header class="header">
+  <h1>todos</h1>
+  <input class="new-todo"
+    autofocus autocomplete="off"
+    placeholder="What needs to be done?"
+    v-model="newTodo"
+    @keyup.enter="addTodo">
+</header>
+----
+
+You can verify that `new-todo` class element contains the expected `placeholder` (input hint)
+with the following test:
+
+[source,js]
+----
+const input = await page.$('input.new-todo'); <1>
```

Comment:
should we have this inside step?

Reply:
After looking at this again, yes, I think you're right.